index.js

```diff
@@ -184,8 +184,7 @@ SonosAccessory.prototype.search = function() {
 
         this.log("Found a playable device at %s for room '%s'", host, roomName);
         this.device = device;
-        search.socket.close(); // we don't need to continue searching.
-
+        search.destroy(); // we don't need to continue searching.
```

Comment:
`search.socket.close()` is incorrect and causes a crash of the plugin after a timeout.